Makefile

```diff
@@ -7,18 +8,17 @@ setup:
 	command -v vbump  || aurget vbump-git
 	command -v pandoc || stack install pandoc
 
-locale/downgrade.pot: downgrade
+locale:
```

Comment:
Please leave these targets how they were. They ensure that the rule is not run if the output file is up to date from the dependencies. What you've created here would be a "phony" though you've not annotated it as such. These are fine to have in addition (see `man`), but not instead of.

Reply:
I had added `locale` as a "phony" below, but I see the point keeping it this way. Adding it back.

Reply:
I understand. Added the original behaviour back, but also added `locale` as a phony so it is easier to call. I also fixed the workflow for the segment with `downgrade.8`, such that the `man` doc is only made if there are changes.

Reply:
The phony annotations are collated on line [41](https://github.com/pbrisbin/downgrade/blob/f330159a4813031a0209628650cade4740012873/Makefile#L41)

Reply:
> The phony annotations are collated on line

Oh, my bad. I forgot I used to do this style. Lately, I've liked putting the `.PHONY` right above the target it's annotating.

Reply:
I will restyle such that the "phony" annotations are right above the targets, so it is easier to see next time.